.github/workflows/tests.yaml

```diff
@@ -47,11 +45,19 @@ jobs:
         python3.10 -m pip install tox
         rm get-pip.py
     
-    - uses: balchua/microk8s-actions@v0.3.2
-      with:
-        channel: '1.28/stable'
-        addons: '["hostpath-storage"]'
+    - name: Install dependencies
+      run: |
+        # Removing docker as it is blocking canonical k8s bootstrap
+        sudo apt-get remove -y docker-ce docker-ce-cli containerd.io
+        sudo rm -rf /run/containerd
+        
+        sudo snap install kubectl --classic
```

Comment:
Do we need this? I see that `kubectl` is a `k8s` command

Reply:
Ok I see that you can't alias it to kubectl 
```
$ sudo snap alias k8s.kubectl kubectl                                                                                                                                                                       1 ↵
error: cannot perform the following tasks:
- Setup manual alias "kubectl" => "kubectl" for snap "k8s" (cannot enable alias "kubectl" for "k8s", target application "kubectl" does not exist)
```
I guess we have to install both snaps